### PR TITLE
add LIS spot test env

### PR DESCRIPTION
**Pull request by lf-zhao**
outcome: now we can run predicators on LIS Spot in its mapped room!

previous PR for adding map in #280 
- map the MIT LIS Spot room, 45-621
- add metadata of the room
- add notes in readme on the steps to add new environment/task for Spot

major actions:
- add a LIS Spot env
- add goal description
- add the env to ground truth factory
- fix unused sim variables for real robot
- tested on LIS Spot, args:
`--spot_robot_ip 192.168.80.3 --spot_graph_nav_map b45-621 --env lis_spot_block_floor_env --approach spot_wrapper[oracle] --bilevel_plan_without_sim True --seed 0 --perceiver spot_perceiver`

### CODE REVIEW

**Comment by lf-zhao** [comment]
should I run some auto reformat tools?

**Comment by lf-zhao** [comment]
updated description!

File: predicators/spot_utils/README.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 # Spot Utils
 
+## How to run your own Spot environment
+
+> Last Updated: 04/11/2024
+
+**Steps:**
+- Set up the codebase, perception pipeline, and Spot
+  - You need to have access to a GPU server for the perception pipeline (e.g., the Detic-SAM pipeline)
+  - You need to connect to Spot (through WiFi or ethernet cable). Spot in its own WiFi AP mode is on IP `192.168.80.3`.
```
Comment (NishanthJKumar) [inline predicators/spot_utils/README.md:10]: This is only true for the LIS spot; so maybe say "Spot in its own WiFi AP mode will have IP address `192.168.80.3`

**Review by NishanthJKumar — APPROVED** [review]
Very nice work @lf-zhao ! Great to see this happen so quickly 💯 
Left one comment. Please run the autoformatting tools specified in the main readme. I will separately take a look at the typechecking errors

File: predicators/spot_utils/README.md (hunk 1 above)
Comment (lf-zhao) [reply predicators/spot_utils/README.md]: done!

**Pull request merged**